bzl/bazel.rc

```diff
@@ -101,3 +101,13 @@ test:asan --test_env=ASAN_ENABLE=1
 test:asan --test_env=ASAN_OPTIONS
 test:asan --test_env=ASAN_SYMBOLIZER_PATH
 test:asan --test_env=DYLD_INSERT_LIBRARIES
+
+# For building tensorflow
+build --action_env=LD_PRELOAD # Needed on *nix to prevent malloc errors in tensorflow
+build --action_env=XLA_DUMPDIR="/tmp/hlo_module"
+build --action_env=XLA_FLAGS=--xla_dump_to="/tmp/hlo_module"
+build --action_env=TF_XLA_FLAGS="--tf_xla_auto_jit=2 --tf_xla_cpu_global_jit --tf_xla_min_cluster_size=0"
```

Comment:
I think having this here will generate & dump the hlo module whenever TF is called via python within the repo, instead of just when necessary for testing. Probably not a real issue but I believe this does make it unnecessary to specify these flags again in the BUILD file